.dockerignore

```diff
@@ -7,3 +7,6 @@ llm/llama.cpp
 .env
 .cache
 test_data
+llm/build
+llama/build
```

Comment:
```suggestion
```

Add this in the `jmorganca/llama` branch?

Reply:
I added it here since it's a bit painful when switching back and forth between branches to have that extra build context, but if you'd prefer to keep it clean I can remove it.